locales-km-KH.xml

```diff
@@ -213,7 +213,7 @@
       <multiple>editors</multiple>
     </term>
     <term name="editorial-director">
-      <single/>
+      <single></single>
```

Comment:
This is odd, but I am leaving it as-is for now in case it was removed for a reason.